Xamarin.Forms.Core/Switch.cs

```diff
@@ -6,11 +6,15 @@ namespace Xamarin.Forms
 	[RenderWith(typeof(_SwitchRenderer))]
 	public class Switch : View, IElementConfiguration<Switch>
 	{
-		public static readonly BindableProperty IsToggledProperty = BindableProperty.Create("IsToggled", typeof(bool), typeof(Switch), false, propertyChanged: (bindable, oldValue, newValue) =>
+		public const string SwitchOnVisualState = "On";
+		public const string SwitchOffVisualState = "Off";
+
+		public static readonly BindableProperty IsToggledProperty = BindableProperty.Create(nameof(IsToggled), typeof(bool), typeof(Switch), false, propertyChanged: (bindable, oldValue, newValue) =>
 		{
 			EventHandler<ToggledEventArgs> eh = ((Switch)bindable).Toggled;
 			if (eh != null)
 				eh(bindable, new ToggledEventArgs((bool)newValue));
```

Comment:
replace with
```csharp
((Switch)bindable).Toggled?.Invoke(bindable, new ToggledEventArgs((bool)newValue));
```

while you're at it

Reply:
Hi @StephaneDelcroix 
Thanks for looking into this. I understand that we are replacing the above code (Line 14 to 16) with the single line code markup as you suggested. This is an existing code.
Quick question (just so I'm aware of before implementing): Is this just only for syntactic sugar or really has any major issue?